src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -282,24 +246,8 @@ export default {
 			return pageViewTrackData;
 		},
 		setRows(data) {
-			if (this.mlServiceBanditExpVersion !== null) {
```

Comment:
As @emuvente surfaced in our chats, this experiment is still active which ensure this query is always run resulting in our bandit algo determining the order of loan channels. This needs to remain in the code as we want to ensure the bandit algo still controls ordering.